study-builder/studies/mpc/snippets/question-therapies-groups.conf

```diff
@@ -0,0 +1,143 @@
+            }
+          ]
+        }
+      ]
+    },
+    "groups": [
+      {include required("picklist-group-question-therapies-hormones.conf")},
+      {include required("picklist-group-question-therapies-chemo.conf")},
+      {include required("picklist-group-question-therapies-other.conf")},
+
+      {
+        "stableId": "EXPERIMENTAL",
+        "nameTemplate": {
+          "templateType": "HTML",
+          "templateText": "$experimental_prompt"
+          "variables": [
+            {
+              "name": "experimental_prompt",
+              "translations": [
+                {
+                  "language": "en",
+                  "text": "EXPERIMENTAL"
+                }
+              ]
+            }
+          ]
+        },
+        "options": [
+          {
+            "stableId": "CLINICAL_TRIAL",
+            "optionLabelTemplate": {
+              "templateType": "TEXT",
+              "templateText": "$experimental_clinical_prompt",
+              "variables": [
+                {
+                  "name": "experimental_clinical_prompt",
+                  "translations": [
+                    {
+                      "language": "en",
+                      "text": "Experiment/Clinical Trial"
+                    }
+                  ]
+                }
+              ]
+            },
+            "detailLabelTemplate": {
+              "templateType": "TEXT",
+              "templateCode": null,
+              "templateText": "$experimental_clinical_details",
+              "variables": [
+                {
+                  "name": "experimental_clinical_details",
+                  "translations": [
+                    {
+                      "language": "en",
+                      "text": "Please provide details"
+                    }
+                  ]
+                }
+              ]
+            },
+            "allowDetails": true,
+            "exclusive": false
+          }
+        ]
+      },
+
+      {
+        "stableId": "OTHER",
+        "nameTemplate": {
+          "templateType": "HTML",
+          "templateText": "$other_prompt"
+          "variables": [
+            {
+              "name": "other_prompt",
+              "translations": [
+                {
+                  "language": "en",
+                  "text": "Other"
+                }
+              ]
+            }
+          ]
+        },
+        "options": [
+          {
+            "stableId": "YES",
+            "optionLabelTemplate": {
+              "templateType": "TEXT",
+              "templateText": "$other_yes_prompt",
+              "variables": [
+                {
+                  "name": "other_yes_prompt",
+                  "translations": [
+                    {
+                      "language": "en",
+                      "text": ""
+                    }
+                  ]
+                }
+              ]
+            },
```

Comment:
This option looks a little odd... I think this shows a checkbox with no text and then there is a "detail" textbox below it. Is that what we're aiming for?

Reply:
Group has only 1 option, Group name label  "Other" will be displayed along with checkbox.

Reply:
Seems a bit weird that the option checkbox itself doesn't have text. I'll take another look once it's deployed to `dev` and see what it looks like.